packages/core/src/components/pagination/pagination.tsx

```diff
@@ -38,6 +38,12 @@ export class AtomPagination {
   makeItems() {
     const items = []
 
+    if (this.count <= 1) {
+      items.push({ type: 'page', number: 1 })
+
+      return items
+    }
```

Comment:
<!-- Thoughts represent an idea that popped up from reviewing. These comments are non-blocking by nature, but they are extremely valuable and can lead to more focused initiatives and mentoring opportunities.  -->
**thought:** ‏I could move this solution into the calc but I thought it would only complicate things, since there is a very simple solution.